src/assignments/scrabble-scorer.rst

```diff
@@ -22,50 +22,173 @@ Requirements
 
 .. admonition:: Note
 
-   The requirements below are what your END assignment will look like.
+   The requirements below are what your assignment should look like when it's 
+   time to submit. Rome wasn't built in a day and neither was Scrabble.
+
    This assignment is broken down so you can complete small pieces as you go.
-   You need to move sequentially starting at part A below. Please read the
-   WHOLE assignment page before starting.
+   You need to move sequentially starting with Part A below. You'll have a much more 
+   enjoyable time writing this program if you read this entire page before even opening repl.it.
 
-#. Create an ``initialPrompt`` function that asks the user which scoring
-   algorithm to use.
-#. Create a ``transform`` function that takes in the ``oldPointStructure``
+#. Write an ``initialPrompt()`` function that asks a user to input a word.
+#. Have the program return a score for the word using ``oldScrabbleScorer()``.
+#. Add additional scoring algorithms and store them in the ``scoringAlgorithms`` array.
+#. Create a ``transform()`` function that takes in the ``oldPointStructure``
    object and returns a ``newPointStructure`` object.
-#. Code three different scoring functions for assigning points to a word.
-#. Create a ``scoringAlgorithms`` array to hold three scoring *objects*. Each
-   object will contain three key/value pairs.
-#. Create a ``runProgram`` function to serve as the starting point for your
+#. Use the ``runProgram()`` function to serve as the starting point for your
    program.
 
+Starter Code
+------------
+
+You only need to pay attention to one file here, ``scrabble-scorer.js``. Within this JavaScript
+file is still some more starter code that you don't need to touch. We'll point out what you 
+should be modifying here to write your Scrabble Scorer program.
+
+Hit the repl.it run button initially and you'll see a message printed to the console:
```

Comment:
Currently, the repl.it starter code still needs updating, but that doesn't prevent the PR from approval.